Modules/_lsprof.c

```diff
@@ -489,14 +489,12 @@ static PyStructSequence_Field profiler_subentry_fields[] = {
 
 static PyStructSequence_Desc profiler_entry_desc = {
     .name = "_lsprof.profiler_entry",
```

Comment:
Hum, I suggest to write `.doc = NULL,` to highlight that these types have no docstring. So maybe tomorrow someone will write a docstring.

Reply:
updated.